app/views/common_parameters/_parameters_table.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="tab-pane" id="params">
+  <div class="tab-content">
+    <ul class="nav nav-tabs" data-tabs="tabs">
```

Comment:
I don't think that an additional level of navigation is helpful here, we now have..

1. application level menus
2. host edit form tabs (Host/Puppet Classes/Parameters etc)
3. Puppet Parameters/Global Parameters
4. list of classes